10.Mender-server/docs.md

```diff
@@ -8,6 +8,98 @@ github: false
 ---
 
 
+## Mender 3.5.1
+
+_Released 03.08.2023_
+
+### Statistics
+
+A total of 279 lines added, 143 removed (delta 136)
+
+| Developers with the most changesets | |
+|---|---|
+| Lluis Campos | 5 (35.7%) |
+| Fabio Tranchitella | 4 (28.6%) |
+| Alf-Rune Siqveland | 2 (14.3%) |
+| Manuel Zedel | 2 (14.3%) |
+| Kristian Amlie | 1 (7.1%) |
+
+| Developers with the most changed lines | |
+|---|---|
+| Fabio Tranchitella | 126 (44.8%) |
+| Kristian Amlie | 63 (22.4%) |
+| Alf-Rune Siqveland | 48 (17.1%) |
+| Lluis Campos | 24 (8.5%) |
+| Manuel Zedel | 20 (7.1%) |
+
+| Top changeset contributors by employer | |
+|---|---|
+| Northern.tech | 14 (100.0%) |
+
+| Top lines changed by employer | |
+|---|---|
+| Northern.tech | 281 (100.0%) |
+
+| Employers with the most hackers (total 5) | |
+|---|---|
+| Northern.tech | 5 (100.0%) |
+
+### Changelogs
+
+#### deployments (4.4.1)
+
+New changes in deployments since 4.4.0:
+
+##### Bug Fixes
+
+* Use separate object storage path for raw file and artifact
+  ([MEN-6091](https://tracker.mender.io/browse/MEN-6091))
```

Comment:
Do we want to keep links to the old Jira instance here?

Reply:
Good catch! No, we don't. I'll fix that, and the tool that generated it as well.